### DISCUSSION TITLE
Differences to original MMG paper

### DISCUSSION THREAD

**Discussion by KVcedric** [ShipMMG/shipmmg#47] [General]
Hello my name is Cedric and I am currently working on the implementation of a maneuvring model.
As I looked for a comparison of my implementations of the 3DOF MMG model I had a look into your code and
observed a difference to the implementations described in "Introduction of MMG standard method for ship maneuvering
predictions". It seems that the motion equations are solved at the center of gravity, although
the paper describes them at the midship location. Besides that the formula used 
for the wake fraction at the propeller plane refers to equation 12 of the paper whose accuracy
seems to be too low according to the authors. Is there a reason you didnt use the suggested equation 16?

**Comment by taiga4112** [comment]
@KVcedric Thank you for good question. Your opinion is correct.
I am using equation 12 simply because the modeling was simple. According to Fig. 10 of the original paper, the value of w_p is highly uncertain. However, we should modify equation 12 because it does not reflect the difference between left and right.

**Reply by KVcedric** [reply]
@taiga4112 Thank you for your quick response. Have you ever tried to recreate the results for the turning maneuver from the paper?
I compared the results from my implementations with yours and they perfectly fit, however so far I did not achieve matching the results from the paper.

**Reply by taiga4112** [reply]
@KVcedric I and other maneuvering researchers (not Japanese) tried to recreate the result for the turning maneuver from this paper. However, we cannot recreate this result perfectly only from the information of this paper....

**Reply by KVcedric** [reply]
@taiga4112 Indeed, there are some information missing e.g. about the propeller position or its rotation rate. Could you tell me if 
the deviations shown in the figure are comparable to your observations or if they are to high?
![MMG_3DOF_KVLCC2_TURN](https://github.com/ShipMMG/shipmmg/assets/38330652/2a9ffe3b-ee40-4158-b0b2-a4c7baabbeb1)

**Reply by taiga4112** [reply]
I think this result is more good as my implementation comparing to MMG paper!!

**Reply by KVcedric** [reply]
Thank you very much for the feedback. It seems you can get even closer to the paper results when reducing the rotation speed of the propeller.
